requirements/edx/base.txt

```diff
@@ -58,7 +58,7 @@ mongoengine==0.7.10
 networkx==1.7
 nltk==2.0.4
 nose==1.3.3
-oauthlib==0.6.3
+oauthlib==0.7.2
```

Comment:
The more I think about it, the more I think this really ought to be done. Can you do that please?